src/main/java/uk/gov/pay/adminusers/service/UserServices.java

```diff
@@ -267,6 +277,12 @@ private void changeUserTelephoneNumber(UserEntity userEntity, String telephoneNu
         userEntity.setUpdatedAt(ZonedDateTime.now(ZoneId.of("UTC")));
         userDao.merge(userEntity);
     }
+    
+    private void changeUserEmail(UserEntity userEntity, String email) {
+        userEntity.setEmail(email);
```

Comment:
I think we'd also want to update the username, which is, for some reason, a different field.

Reply:
Uh why? We're not updating the username but the email address

Reply:
[Fixed](https://github.com/alphagov/pay-adminusers/pull/1229/commits/c071c9dbab19d7f7e0fab1c2c887efd023d5b874)